templates/app.php

```diff
@@ -1,3 +1,6 @@
 <div id="wpquerypush">
     <h2><?php esc_html_e( 'WPQueryPush Loading', 'WPQueryPush' ); ?>...</h2>
 </div>
+<div id="wpquerypush-nonce">
+    <?php wp_nonce_field( 'wpquerypush' ); ?>
```

Comment:
@prykon Is this custom nonce being validated anywhere? (I didn't see it, and don't seem to be able to search the PR)

Reply:
Yes, in rest-api/rest-api.php, line 8.